src/Plot/Scale.elm

```diff
@@ -0,0 +1,32 @@
+module Plot.Scale where
+
+import Private.Scale.Linear as LinearScale
+import Private.Scale.OrdinalBands as OrdinalBands
+import Private.Extras.Set as Set exposing (Set)
+import Private.Scale exposing (Scale)
+
+type alias LinearScale = Scale Set Float
```

Comment:
The `LinearScale` type is public and will be exposed to users where as the methods from `Private.Scale.Linear` are only used internally. To achieve this they need to be in separate files.